packages/wrangler/src/config/config.ts

```diff
@@ -183,6 +185,18 @@ export interface ConfigFields<Dev extends RawDevConfig> {
 	keep_vars?: boolean;
 }
 
+// Pages-specific configuration fields
+export interface PagesConfigFields {
+	/**
+	 * The directory of static assets to serve.
+	 *
+	 * The presence of this field in `wrangler.toml` indicates a Pages project,
+	 * and will prompt the handling of the configuration file according to the
+	 * Pages-specific validation rules.
+	 */
+	pages_build_output_dir: string | undefined;
```

Comment:
```suggestion
	pages_build_output_dir?: string;
```

to avoid people having to specify it when creating an object of this type in programmatic scenarios.